src/foam/nanos/theme/Theme.js

```diff
@@ -119,6 +119,11 @@ foam.CLASS({
         ]
       },
     },
+    {
+      name: 'acceptanceDocumentsCapability',
```

Comment:
just thinking that this capability should be named to be something like 'userRegistrationCapability'